lib/redis.utils.ts

```diff
@@ -10,6 +10,7 @@ export function createRedisConnection(
   if (connection?.url) {
     return new Redis(connection?.url, connection);
   } else {
-    return new Redis(connection);
+    // TODO appropriate verification is required when the connection is undefined.
```

Comment:
No need for manual verification. The undeling lib, ioredis, will use the default connection params, or will not establish a connection, and the strategy will not start.